core/lib/permissions.ts

```diff
@@ -18,3 +18,39 @@ export const getPubUsers = (permissions: PermissionPayload[]) => {
 	}
 	return users;
 };
+
+enum Privileges {
+	movePub,
+	createPub,
+	viewPub,
```

Comment:
`archive/deletePub` as well right?
also thanks for including a separate `viewPub` privilege. do you think we should also add this to other places? for instance, in #656 @gabestein  describes the `contributor` role for a stage as 

> "Can see Stages they have been directly added to, and the name of stages any Pubs they have been added to are in".

this implies that they can only see the stage, not edit it (which i think is good). So would we need a separate `viewStage` privilege, or do you that is already captured some other way?

Reply:
Yes to deletePub (maybe archive in the future if we build that).

For stage viewing, I had been thinking that a `viewPub` privilege would be enough, since it's the pubs in the stage you want to see, not the stage, but I think you're right!